src/spine.js

```diff
@@ -540,7 +569,9 @@ pc.extend(pc, function () {
 
     Spine.prototype.SubmitBatch = function (indexBase, indexCount, materialKey) {
         if (indexCount > 0) {
-            var mesh = new pc.Mesh();
+            // assumes Engine version 1.27+
```

Comment:
I don't think you need this comment. Even if an engine before 1.27.0 was used (which is pretty unlikely!), it'd still be valid code because the parameter would simply be ignored.

Reply:
Completed in: 694e416